Source/Meadow.Contracts/Hardware/Networking/CellNetwork.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Meadow.Networking;
+
+public record CellNetwork
+{
+    public int Index { get; set; }
```

Comment:
As mentioned [here](https://github.com/WildernessLabs/Meadow.Core/pull/347/files#r1279675212), I suggest you change the `Index` to another name, maybe `State`, or something like that.